vertical-pod-autoscaler/hack/tag-release.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Copyright 2025 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Convenience script that tags the VPA automatically built images with a
+# version tag in preparation for release.
+
+# The following environment variables must be set for this script:
+# BUILD_TAG: corresponds to the tag of the automatic build (eg. v20250512-cluster-autoscaler-chart-9.46.6-151-g2b33c4c79)
+# TAG: corresponds to the version of the release (eg. 1.4.0)
+
+# See VPA release instructions (in RELEASE.md) before using this file.
+
+if [[ -z "$BUILD_TAG" ]]; then
+    echo "BUILD_TAG must be set to the existing image build tag (eg. BUILD_TAG=v20250512-cluster-autoscaler-chart-9.46.6-151-g2b33c4c79)"
+    exit 1
+fi
+if [[ -z "$TAG" ]]; then
+    echo "TAG must be set to the VPA release number (eg. TAG=1.4.0)"
+    exit 1
+fi
+
+echo "Adding tag $TAG based on built tag $BUILD_TAG"
+
+read -p "Press enter to continue"
+
+gcloud container images add-tag -q gcr.io/k8s-staging-autoscaling/vpa-admission-controller:$BUILD_TAG gcr.io/k8s-staging-autoscaling/vpa-admission-controller:$TAG --project=k8s-staging-autoscaling
```

Comment:
It may be worth adding checks to see if the `gcloud` binary is installed and logged in correctly, but that can happen later. This script is already making the release process better, so happy to merge it.